appcelerator-archive/amp
# dynamic gateway protobuf targets

fixes #620 

- Makefile target list for gateway protobuf target is now dynamic

to test:
- make clean
- make proto
- git status # workspace should be clean
- make install
